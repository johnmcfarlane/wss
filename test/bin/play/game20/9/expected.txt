 49  8,2  - FOX
 46  8,2  - YOX
 41  9,2  - OXO
 36  9,2  - OX
 36 10,2  | XI
 33  4,2  | COLFOX
 32  6,5  - FEIL
 30  2,2  | XYLIA
 29  8,2  - FOY
 28  4,6  | OXO
 27  5,11 - FCY
 27  1,7  - FLEX
 26  2,2  | LOXIA
 26  5,8  - OX
 26  8,7  | OXO
 25 11,5  | COX
 25  7,5  - EYL
 25 11,5  | FIX
 25  8,2  - FOL
 25 11,5  | FOX
 25  9,2  - OFO
 25  2,7  - YEX
 24  4,4  | CLOF
 24  8,3  | ELF
 24  1,7  - ILEX
 24  8,2  - OOF
 24 11,5  | YOX
 23  4,4  | COOF
 23  2,7  - LEX
 23 11,5  | LOX
 22  4,5  | COX
 22  8,8  | COY
 22  4,4  - FIXE
 22  4,5  | FOX
 22  8,8  | FOY
 21  1,9  - CIR
 21  4,3  | CLOOF
 21  1,9  - COR
 21  1,9  - FIR
 21  8,2  - FOO
 21  1,9  - FOR
 21  5,11 - ICY
 21  9,6  | IF
 21  9,6  | OF
 21 11,6  | OX
 21  6,8  - XI
 21 11,7  | XIs
 21  7,1  - XYZ
 21  4,5  | YOX
 20  8,7  | CLOY
 20  1,11 - COP
 20 10,2  | FI
 20  1,11 - FOP
 20  4,4  - FYCE
 20  4,5  | LOX
 20  9,2  - OF
 20  4,6  | OFO
 20  6,8  - OIL
 20  3,11 - POY
 20  4,3  | XYLO
 19  1,7  - CLEF
 19  4,4  | COOL
 19  4,4  | FOOL
 18 11,1  | CFD
 18  1,8  - COWY
 18  7,1  - COZY
 18  3,7  - EX
 18  4,4  - FICE
 18  2,2  | FOLIA
 18  7,1  - FOZY
 18  4,6  | OX
 18  3,11 - POL
 18  1,9  - YOR
 17  8,7  - FLEX
 17  1,7  - FLEY
 17  3,4  - FOXIE
 17  4,4  | LOOF
 17  9,6  | OY
 17  9,1  - ZOL
 17  9,1  | ZORIL
 16  5,3  | CLIV
 16  4,4  - COLE
 16  4,3  | COYOL
 16 11,1  | FIDO
 16  4,4  - FILE
 16 11,1  | FLD
 16  4,4  - FLOE
 16  4,4  - FOLE
 16  1,8  - LOWY
 16  8,8  | LOY
 16  9,2  - OY
 16  5,8  - OY
 16  3,11 - POI
 16  6,5  - YEX
 16  6,5  | YEX
 16 10,2  | YI
 15 11,1  | CID
 15  4,4  | CLO
 15 11,1  | COD
 15  7,11 - COQ
 15  1,8  - COWL
 15  7,1  - COZ
 15  8,3  | EL
 15  7,4  - ELI
 15 11,1  | FID
 15  7,1  - FIZ
 15  4,4  | FLO
 15 11,1  | FOD
 15  1,8  - FOWL
 15  1,7  - LIEF
 15 11,5  | LIF
 15  1,9  - LIR
 15 11,5  | LOF
 15  1,9  - LOR
 15  4,4  | OLOF
 15  4,4  - YCIE
 15  9,7  - YEX
 14  2,5  | CAF
 14  2,4  | CLAY
 14  4,4  | COO
 14  2,3  | COXA
 14  6,5  - FEC
 14  6,5  | FEC
 14  2,4  | FLAY
 14  4,4  | FOO
 14  8,7  - ILEX
 14  1,11 - LOP
 14  8,7  | OFO
 14  5,5  | XVI
 14  2,3  | XYLA
 14  1,8  - YOWL
 13  5,4  | CIVY
 13  8,7  - CLEF
 13 11,5  | COL
 13  9,7  - FEC
 13  6,5  - FEY
 13  6,5  | FEY
 13 11,5  | FIL
 13  6,4  | FOE
 13 11,5  | FOL
 13  4,2  | FOLIO
 13 11,6  | IF
 13  9,7  - LEX
 13  2,7  - LEY
 13  7,11 - LIQ
 13  7,1  - LIZ
 13  2,4  | LOAF
 13  7,11 - LOQ
 13 11,6  | OF
 13  2,4  | OLAF
 12  2,5  | CAY
 12  6,6  | EXIES
 12  2,5  | FAY
 12  9,7  - FEY
 12  4,3  | FICO
 12  8,7  - FLEY
 12 11,1  | LCD
 12  4,4  - LICE
 12  4,4  - LIFE
 12  4,5  | LOF
 12  4,4  | OXO
 12 11,5  | XI
 12 11,1  | YOD
 11  8,7  | CLO
 11  8,7  - COEF
 11  4,3  | COYO
 11  2,8  - CWO
 11  2,3  | FILAO
 11  8,7  | FLO
 11  3,4  - FOLIE
 11  2,5  | OAF
 11  4,5  | OOF
 11  2,8  - OWL
 11  6,8  - YI
 10 11,2  | CDF
 10  9,6  - CDF
 10  5,4  | CIV
 10  4,5  | COL
 10  3,7  - EF
 10  6,5  - FE
 10  6,5  | FE
 10  2,3  | FICA
 10  4,3  | FILO
 10  4,5  | FOL
 10  8,7  - ICE
 10 10,3  | IF
 10  9,4  - IF
 10  1,5  - IF
 10  8,7  - IFE
 10  7,8  - IO
 10  5,9  | KAF
 10  2,5  | LAY
 10 11,1  | LIDO
 10  4,3  | LIFO
 10  4,3  | LOCO
 10  1,7  - LYE
 10  1,5  - OF
 10  4,6  | OF
 10 11,5  | OIL
 10  5,8  | OKAY
 10 11,1  | OLDY
 10  8,9  | OY
 10  6,8  | XES
 10  8,8  | YO
 10  8,10 - YUS
  9  2,4  | CIAO
  9  1,8  - COW
  9 11,3  | DILO
  9 11,3  | DOLI
  9  7,5  - EX
  9  6,6  | EX
  9 10,7  - EX
  9  9,7  - FE
  9  8,7  - FIEL
  9  9,6  | IO
  9  5,5  | IVY
  9  5,9  | KAY
  9  2,7  - LEI
  9 11,1  | LID
  9  8,7  - LIEF
  9 11,7  | LIs
  9 11,1  | LOD
  9  8,7  - LYE
  9  4,3  | OLIO
  8  2,6  | AY
  8  1,7  - CIE
  8  2,3  | COLA
  8  4,5  | COO
  8  8,7  | COO
  8 11,5  | FI
  8  1,7  - FIE
  8  2,3  | FILA
  8  4,5  | FOO
  8  8,7  | FOO
  8 11,2  | IDLY
  8 11,2  | IDYL
  8  5,9  | KAL
  8  9,7  - LEY
  8  5,4  | LIV
  8  2,3  | LOCA
  8  4,4  | LOO
  8  2,10 | NO
  8  4,10 | NO
  8 11,2  | ODIC
  8 11,2  | ODYL
  8  4,4  | OFO
  8 10,2  | OI
  8  9,2  - OO
  8  2,11 - OP
  8  1,5  - OY
  8  3,11 - PO
  8  8,2  | YE
  8  6,5  - YE
  8  6,5  | YE
  8  2,7  - YE
  8  8,2  - YO
  8  1,8  - YOW
  7  8,7  - CIE
  7  8,7  - COE
  7 11,3  | DIF
  7 11,3  | DOC
  7  8,7  - FIE
  7  8,7  - FOE
  7  1,7  - ILE
  7  2,3  | ILYA
  7 11,6  | IO
  7  5,9  | KAI
  7  8,8  | LO
  7  1,8  - LOW
  7  6,8  - OI
  7  5,8  | OKA
  7 11,1  | OLD
  7  1,7  - OLE
  7  3,8  - WO
  7  9,7  - YE
  7 11,5  | YI
  7 11,5  | YO
  6  2,4  | CIA
  6  5,4  - CIE
  6  5,4  - COE
  6  6,9  | ESC
  6  5,4  - FIE
  6  5,4  - FOE
  6  5,4  - ICE
  6 11,2  | IDOL
  6  5,4  - IFE
  6  8,7  - ILE
  6  2,5  | LAO
  6  9,7  - LEI
  6 11,5  | LI
  6  1,7  - LIE
  6 11,5  | LO
  6  4,5  | LOO
  6  8,7  | LOO
  6  5,4  - LYE
  6  2,4  | OCA
  6  8,7  - OLE
  6  6,8  | YESO
  5 11,3  | DIL
  5 11,3  | DOL
  5  7,5  - EF
  5  6,6  | EF
  5 10,7  - EF
  5  2,5  | FA
  5  6,4  - FE
  5  2,5  | IAO
  5 11,5  | IO
  5  8,7  - LIE
  5  8,7  - LOE
  5 11,5  | OI
  5  2,9  | ON
  5  2,9  - OR
  5  2,8  - OW
  5  6,8  | YES
  4  3,7  - EO
  4  2,4  | ILA
  4  5,4  - ILE
  4 10,3  | IO
  4  9,4  - IO
  4  8,8  | IO
  4 11,8  | Is
  4 10,8  - LI
  4  5,4  - LIE
  4 10,8  - LO
  4  2,4  | LOA
  4  5,4  - LOE
  4  8,2  | OE
  4  6,5  - OE
  4  6,5  | OE
  4  2,4  | OLA
  4  5,4  - OLE
  4  8,2  - OO
  4  4,6  | OO
  4  8,8  | OO
  4 11,8  | Os
  4  2,5  | YA
  4  6,4  - YE
  4  4,5  | YO
  3 11,3  | DI
  3 10,6  - DI
  3 11,3  | DO
  3 10,6  - DO
  3  7,5  - EL
  3  6,6  | EL
  3 10,7  - EL
  3 11,2  | ID
  3  9,6  - ID
  3  2,5  | LA
  3  4,5  | LO
  3 11,2  | OD
  3  9,6  - OD
  3  9,7  - OE
  3  6,8  | OES
  2  7,5  - EO
  2  6,6  | EO
  2 10,7  - EO
  2  4,5  | IO
  2  7,8  | IT
  2  6,4  - OE
  2  4,5  | OO
