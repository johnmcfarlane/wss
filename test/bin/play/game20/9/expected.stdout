 49  8,2  - FOX
 41  9,2  - OXO
 36  9,2  - OX
 36 10,2  | XI
 29  8,2  - FOY
 28  4,6  | OXO
 27  1,7  - FLEX
 26  5,8  - OX
 26  8,7  | OXO
 25 11,5  | COX
 25 11,5  | FIX
 25 11,5  | FOX
 24  1,7  - ILEX
 23  4,4  | COOF
 23  2,7  - LEX
 23 11,5  | LOX
 22  4,5  | COX
 22  4,5  | FOX
 21  1,9  - COR
 21  1,9  - FIR
 21  1,9  - FOR
 21  9,6  | IF
 21  9,6  | OF
 21 11,6  | OX
 21  6,8  - XI
 21 11,7  | XIs
 20  1,11 - COP
 20 10,2  | FI
 20  1,11 - FOP
 20  4,4  - FYCE
 20  4,5  | LOX
 20  9,2  - OF
 20  6,8  - OIL
 19  1,7  - CLEF
 19  4,4  | COOL
 19  4,4  | FOOL
 18  1,8  - COWY
 18  7,1  - COZY
 18  3,7  - EX
 18  4,4  - FICE
 18  2,2  | FOLIA
 18  7,1  - FOZY
 18  4,6  | OX
 17 11,5  | FILOs
 17  8,7  - FLEX
 17  1,7  - FLEY
 17  4,4  | LOOF
 17  9,6  | OY
 17  9,1  | ZORIL
 16  4,4  - COLE
 16 11,1  | FIDO
 16  4,4  - FILE
 16  4,4  - FLOE
 16  9,2  - OY
 16  5,8  - OY
 16  3,11 - POI
 16 10,2  | YI
 15 11,1  | COD
 15  7,11 - COQ
 15  1,8  - COWL
 15  7,1  - COZ
 15 11,1  | FID
 15  7,1  - FIZ
 15  1,8  - FOWL
 15  1,7  - LIEF
 14  2,4  | CLAY
 14  4,4  | COO
 14  2,3  | COXA
 14  2,4  | FLAY
 14  8,7  - ILEX
 14  1,11 - LOP
 14  1,8  - YOWL
 13  8,7  - CLEF
 13 11,5  | COL
 13  6,5  - FEY
 13  6,5  | FEY
 13 11,5  | FIL
 13  6,4  | FOE
 13  4,2  | FOLIO
 13 11,6  | IF
 13  9,7  - LEX
 13  2,7  - LEY
 13  2,4  | LOAF
 13  7,11 - LOQ
 13 11,6  | OF
 12  2,5  | CAY
 12  2,5  | FAY
 12  9,7  - FEY
 12  4,3  | FICO
 12  8,7  - FLEY
 12  4,4  - LICE
 12  4,4  - LIFE
 12  4,4  | OXO
 12 11,5  | XI
 12 11,1  | YOD
 11  9,7  - CEL
 11  2,5  | OAF
 11  2,8  - OWL
 11  6,8  - YI
 10  4,5  | COL
 10  3,7  - EF
 10  6,5  - FE
 10  6,5  | FE
 10  4,3  | FILO
 10  8,7  - ICE
 10 10,3  | IF
 10  9,4  - IF
 10  1,5  - IF
 10  5,9  | KAF
 10  2,5  | LAY
 10 11,1  | LIDO
 10  4,3  | LOCO
 10  1,7  - LYE
 10  1,5  - OF
 10  4,6  | OF
 10 11,5  | OIL
 10  5,8  | OKAY
 10 11,1  | OLDY
 10  6,8  | XES
 10  8,8  | YO
  9  2,4  | CIAO
  9  1,8  - COW
  9  7,5  - EX
  9  6,6  | EX
  9 10,7  - EX
  9  9,7  - FE
  9  5,5  | IVY
  9  5,9  | KAY
  9  2,7  - LEI
  9 11,1  | LID
  9  8,7  - LIEF
  9 11,7  | LIs
  9  8,7  - LYE
  9  4,3  | OLIO
  8  2,6  | AY
  8  2,3  | COLA
  8  4,5  | COO
  8  8,7  | COO
  8 11,5  | FI
  8  1,7  - FIE
  8  2,3  | FILA
  8 11,2  | IDLY
  8 11,2  | IDYL
  8  9,7  - LEY
  8  2,3  | LOCA
  8  4,4  | LOO
  8  2,10 | NO
  8  4,10 | NO
  8 11,2  | ODIC
  8 11,2  | ODYL
  8 10,2  | OI
  8  9,2  - OO
  8  2,11 - OP
  8  1,5  - OY
  8  3,11 - PO
  8  8,2  | YE
  8  6,5  - YE
  8  6,5  | YE
  8  2,7  - YE
  8  8,2  - YO
  8  1,8  - YOW
  7 11,3  | DIF
  7 11,3  | DOC
  7  8,7  - FIE
  7  8,7  - FOE
  7  5,9  | KAI
  7  8,8  | LO
  7  1,8  - LOW
  7  6,8  - OI
  7  5,8  | OKA
  7 11,1  | OLD
  7  1,7  - OLE
  7  3,8  - WO
  7  9,7  - YE
  7 11,5  | YI
  7 11,5  | YO
  6  5,4  - FIE
  6  5,4  - FOE
  6  5,4  - ICE
  6 11,2  | IDOL
  6  9,7  - LEI
  6 11,5  | LI
  6  1,7  - LIE
  6 11,5  | LO
  6  4,5  | LOO
  6  8,7  | LOO
  6  5,4  - LYE
  6  2,4  | OCA
  6  8,7  - OLE
  5 11,3  | DOL
  5  7,5  - EF
  5  6,6  | EF
  5 10,7  - EF
  5  2,5  | FA
  5  6,4  - FE
  5  8,7  - LIE
  5 11,5  | OI
  5  2,9  | ON
  5  2,9  - OR
  5  2,8  - OW
  5  6,8  | YES
  4  3,7  - EO
  4 11,8  | Is
  4 10,8  - LI
  4  5,4  - LIE
  4 10,8  - LO
  4  8,2  | OE
  4  6,5  - OE
  4  6,5  | OE
  4  5,4  - OLE
  4  8,2  - OO
  4  4,6  | OO
  4  8,8  | OO
  4 11,8  | Os
  4  2,5  | YA
  4  6,4  - YE
  4  4,5  | YO
  3 11,3  | DI
  3 10,6  - DI
  3 11,3  | DO
  3 10,6  - DO
  3  7,5  - EL
  3  6,6  | EL
  3 10,7  - EL
  3 11,2  | ID
  3  9,6  - ID
  3  2,5  | LA
  3  4,5  | LO
  3 11,2  | OD
  3  9,6  - OD
  3  9,7  - OE
  3  6,8  | OES
  2  7,5  - EO
  2  6,6  | EO
  2 10,7  - EO
  2  7,8  | IT
  2  6,4  - OE
  2  4,5  | OO
